feeds/wlan-ap/opensync/files/bin/wlan_ap_redirector.sh

```diff
@@ -1,11 +1,30 @@
 #!/bin/sh
 
-if [ $# -ne 1 ] ; then
-	echo "Usage: $0 <redirector address>" >&2
-	exit 1
+AP_PRIVATE_KEY_FILE="/usr/opensync/certs/client_dec.key"
+AP_CERTIFICATE_FILE="/usr/opensync/certs/client.pem"
+AP_DEVICE_ID_FILE="/usr/opensync/certs/deviceid.txt"
+DIGICERT_API_URI="clientauth.demo.one.digicert.com"
```

Comment:
Is this URL correct?

Reply:
TIP Root signed certificates must exist for all AP NOS / TIP NOS based devices. 

These function in mTLS therefore on each device 3 files are present:

device key

device certificate

CA certificate

This ensures a device is unique (key) is signed from same chain (device cert) and can validate the TLS handshake with server peer ( CA cert ). 

Note:

Device PKI files notes above MUST survive a factory reset 

Device MUST implement DigiCert device API lookup for current value of ‘redirector’ 
https://dev.digicert.com/digicert-one/iot-device-manager/iot-devices-v2/get-device-details-client-auth/ 

Which indicates the endpoint as: GET https://clientauth.one.digicert.com/iot/api/v2/device/{{device_id}}